katportalclient/test/test_client.py

```diff
@@ -1565,44 +1722,86 @@ def mock_async_fetchers(valid_responses, invalid_responses, starts_withs=None,
     assert(len(starts_withs) == num_calls)
```

Comment:
`mock_async_fetchers` must be changed to a class method - see code in master branch.